test-integration/helpers/reading-builder.coffee

```diff
@@ -269,10 +273,11 @@ class ReadingBuilder extends TestHelper
           @cancel() if isDisplayed
       else
         # Click "Add Readings"
+        @test.sleep(100) # Not sure why this is needed
         @el.addReadingsButton.click()
```

Comment:
Yeah, I *think* that works for some of them.  It's hard to debug though since the timeouts are pretty transient.  A 100ms timeout also works and is pretty painless.